static/src/javascripts/projects/common/modules/ui/subscription-banners/guardian-weekly-banner-template.js

```diff
@@ -0,0 +1,115 @@
+// @flow
+
+import closeCentralIcon from 'svgs/icon/close-central.svg';
+import theGuardianLogo from 'svgs/logo/the-guardian-logo.svg';
+
+const isUserLoggedIn = userLoggedIn =>
+    userLoggedIn
+        ? 'site-message--subscription-banner__sign-in--already-signed-in'
+        : '';
+
+const subscriptionBannerTemplate = (
+    subscriptionUrl: string,
+    signInUrl: string,
+    userLoggedIn: boolean
+): string => `
+<div id="js-subscription-banner-site-message" class="site-message--subscription-banner site-message--gw-banner">
+    <div class="site-message--subscription-banner__inner">
+        <h3 class="site-message--subscription-banner__title">
+           Read The Guardian in Print
+        </h3>
+
+        <div class="site-message--subscription-banner__description">
+            <p>Support The Guardian's independent journalism by subscribing
+             to The Guardian Weekly, our essential world news magazine</p>
+        </div>
+
+        <div class="site-message--subscription-banner__cta-container">
+            <a
+                id="js-site-message--subscription-banner__cta"
+                class="site-message--subscription-banner__cta"
+                data-link-name="subscription-banner : cta"
+                href="${subscriptionUrl}"
+
+            >
+                <span class="cta-text-mobile">Subscribe Now</span>
+                <span class="cta-text-desktop">Become a Guardian weekly Subscriber</span>
+            </a>
+            <div class="site-message--subscription-banner__cta-dismiss-container ${isUserLoggedIn(
+                userLoggedIn
+            )}">
+                <a
+                    id="js-site-message--subscription-banner__cta-dismiss"
+                    class="site-message--subscription-banner__cta-dismiss"
+                    data-link-name="subscription-banner : not now"
+                    tabindex="0"
+                >
+                    Not now
+                </a>
+            </div>
+        </div>
+
+        <div class="site-message--subscription-banner__sign-in ${isUserLoggedIn(
+            userLoggedIn
+        )}"
+        >
+            <p>Already a subscriber?</p>
+            <br class="temp-mobile-break">
+            <a
+                id="js-site-message--subscription-banner__sign-in"
+                class="site-message--subscription-banner__subscriber-link"
+                data-link-name="subscription-banner : sign in"
+                href="${signInUrl}"
+
+            >
+                <span class="site-message--subscription-banner__sign-in-link">Sign in</span> to not see this again
```

Comment:
The message "Sign in to not see this again" is not completely accurate since if a user logs off they'll see the message again. I'm wondering if a message like "Sign in to stop seeing this message" would be more clear. Question for UX really.